<launch>
  <node name="sweep_scan"	pkg="sweep_ros"  type="sweep_scan" output="screen">
  	<param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
  	<param name="serial_baudrate"     type="int"    value="115200"/>
  	<param name="frame_id"            type="string" value="laser_frame"/>
  	<param name="rotation_speed"	  type="int" value="10"/>
  </node>
</launch>
